script/lib.sh

```diff
@@ -11,7 +11,7 @@ get_branch() {
   # See http://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables
   # for details about default Travis Environment Variables and their values
   if [ -z "${TRAVIS_BRANCH-}" ]; then
-    BRANCH=$(git symbolic-ref HEAD | sed -e 's,.*/\(.*\),\1,')
+    BRANCH=$(git rev-parse --abbrev-ref HEAD)
```

Comment:
This has two advantages:

1. shorter and look ma, no `sed` ;)
2. better behavior when not on a branch (`HEAD` vs some random error that I think sed will eat)

Works in git 1.7+, which is like from 2010.